.env.test

```diff
@@ -7,3 +7,4 @@ POSTGRES_PORT=54320
 POSTGRES_USER=root
 SLACK_BOT_USER_O_AUTH_ACCESS_TOKEN=SLACK_BOT_USER_O_AUTH_ACCESS_TOKEN
 SLACK_SIGNING_SECRET=SLACK_SIGNING_SECRET
+EMAIL_PATTERNS=@manomano.com,@prt.manomano.com
```

Comment:
Please configure your IDE to always add a newline at end of file, this will make the red pictogram disappear.